### PR TITLE
0.6.0 release fixes

### PR DESCRIPTION
**Pull request by zah**
* Updated README
* No double v in nimbus_beacon_node --version
* No md5 checksums

### CODE REVIEW

File: docker/dist/README.md [hunk 1]
```diff
@@ -25,35 +25,39 @@ No `-march=native` and no metrics support.
 With default options:
 
 ```bash
-./run_medalla_node.sh
+./run-medalla-node.sh
 ```
```
Comment (stefantalpalaru) [inline docker/dist/README.md:29]: Should be "run-medalla-beacon-node.sh".

File: docker/dist/README.md [hunk 2]
```diff
@@ -25,35 +25,39 @@ No `-march=native` and no metrics support.
...
 
 ```bash
-GOERLI_WEB3_URL="wss://goerli.infura.io/ws/v3/<YOUR PROJECT ID>" ./run_medalla_node.sh
+WEB3_URL="wss://goerli.infura.io/ws/v3/<YOUR PROJECT ID>" ./run-medalla-node.sh
 ```
 
 ## Running a mainnet node
 
-Same conventions as the Medalla script described above, plus the requirement of specifying a Web3 URL:
+`run-mainnet-node.sh` is a similar script intended for connecting to mainnet.
```
Comment (stefantalpalaru) [inline docker/dist/README.md:54]: Same: "run-mainnet-beacon-node.sh"

File: docker/dist/README.md (hunk 1 above)
Comment (zah) [reply docker/dist/README.md:29]: Right, thanks

File: docker/dist/README.md [hunk 3]
```diff
@@ -25,35 +25,39 @@ No `-march=native` and no metrics support.
 With default options:
 
 ```bash
-./run_medalla_node.sh
+./run-medalla-beacon-node.sh
 ```
 
-Change options implemented as shell variables inside the script:
+The script will forward all supplied options to the `nimbus_beacon_node` executable:
```
Comment (stefantalpalaru) [inline docker/dist/README.md:31]: "nimbus_beacon_node_spec_0_12_3"

**Review by stefantalpalaru — APPROVED** [review]

**Pull request merged**